docs/pages/guides/keyboard-handling.mdx

```diff
@@ -0,0 +1,287 @@
+---
+title: Keyboard handling
+description: A guide for handling common keyboard interactions.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
```

Comment:
```suggestion
import { GithubIcon } from '@expo/styleguide-icons/custom/GithubIcon';

import { BoxLink } from '~/ui/components/BoxLink';
import { Terminal } from '~/ui/components/Snippet';
import { CODE } from '~/ui/components/Text';
```

Let's add these import statements so that we can add BoxLinks in Additiona resources section.